controller/Review.php

```diff
@@ -109,7 +121,9 @@ public function index(): void
             'execution' => $execution->getIdentifier(),
             'delivery' => $delivery->getUri(),
             'show-score' => (int) $finder->getShowScoreOption($launchData),
-            'show-correct' => (int) $finder->getShowCorrectOption($launchData)
+            'show-correct' => (int) $finder->getShowCorrectOption($launchData),
```

Comment:
Not so sure why we need to convert the boolean to int here. Can't we use it as is in the template?

Reply:
The filter_var function will handle it, in fact, the method `getShowCorrectOption` will always return a boolean value. I'm just a bit confused why we're converting it to integer when assigning to the template

Reply:
Probably could indeed. I'm just allowing the previous developer's choice to stand, to minimise changes.